BREAKING_CHANGES.md

```diff
@@ -4,10 +4,91 @@ This is a comprehensive list of the breaking changes introduced in the major ver
 
 ## Versions
 
+- [Stencil 4.x](#stencil-v400)
 - [Stencil 3.x](#stencil-v300)
 - [Stencil 2.x](#stencil-two)
 - [Stencil 1.x](#stencil-one)
 
+## Stencil v4.0.0
+
+### General
```

Comment:
Note there's no table of contents. In fact, I've broken the one for v3 (there are two 'general' anchors). I need to work through how this ought to look, but don't wanna/think that should be a blocker